backend/common/pom.xml

```diff
@@ -116,6 +116,16 @@
             <artifactId>log4j-slf4j-impl</artifactId>
             <version>2.15.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>2.15.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+            <version>2.15.0</version>
```

Comment:
Nit: let's put this in a variable already.

Reply:
Extracted in variable